### PR TITLE
Flipping page

### PR DESCRIPTION
**Pull request by sajedazoabi**
relates #26  - but it still flipping without clicking the info button

### CODE REVIEW

File: src/containers/career.jsx [hunk 1]
```diff
@@ -33,19 +32,26 @@ class Career extends Component {
...
+      <CareerInfo career={this.props.careers[1]}/>
+      <button id='like' onClick={() => {
+        this.nextCareer();
+      }}>LIKE</button>
+      <button id='dislike' onClick={() => {
+        this.nextCareer();
+      }}>DISLIKE</button>
+		</div>
+	</div>
+</div>
```
Comment (matthewdking) [inline src/containers/career.jsx]: Indentation here needs fixing

File: public/style.css [hunk 2]
```diff
@@ -0,0 +1,54 @@
+/* entire container, keeps perspective */
+.flip-container {
+	perspective: 1000px;
+}
+/* flip the pane when hovered */
+/*.flip-container:hover .flipper, .flip-container.hover .flipper {
```
Comment (Karyum) [inline public/style.css]: whats this ?

File: public/style.css [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+}*/
+
+.flip {
+
+	transform: rotateY(180deg);
+}
+
+.flip-container, .front, .back {
+	width: 320px;
```
Comment (Karyum) [inline public/style.css]: dont use pixels trust me its not going to end up well lol

File: public/style.css [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+	transform-style: preserve-3d;
+	position: relative;
+}
+
+/* hide back of pane during swap */
+.front, .back {
+	backface-visibility: hidden;
+	position: absolute;
+	top: 0;
+	left: 0;
```
Comment (Karyum) [inline public/style.css]: indentation

File: public/style.css [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+	transform: rotateY(0deg);
+}
+
+/* back, initially hidden pane */
+.back {
+	transform: rotateY(180deg);
+}
+.btn{
+position:absolute;
+bottom:30%;
```
Comment (Karyum) [inline public/style.css]: indentation

File: src/containers/career.jsx [hunk 6]
```diff
@@ -69,34 +77,40 @@ class Career extends Component {
...
-            id='swipeZone'
-            career={this.props.careers[this.state.currentIndex]}
-          />
+        <div className="flip-container">
+          <div id="swipeZone" className="flipper"
+            onTouchStart={event => handleTouchStart(event)}
+            onTouchEnd={event => this.handleTouchEnd(event)}>
+            <div className="front">
+              <CareerCard career={this.props.careers[this.state.currentIndex]}/>
+              <button onClick={() => {
```
Comment (Karyum) [inline src/containers/career.jsx]: onClick should on a new line 
theres more :arrow_down:

**Pull request merged**